data/tlg0341/tlg002/tlg0341.tlg002.perseus-grc1.xml

```diff
@@ -1,1606 +1,1604 @@
+                <l n="906">Γόννον Φάλαννάν τʼ ἠδʼ Ὀλοσσόνων γύας</l>
+                <l n="907">καὶ Κασταναίαν, ἀκτέριστον ἐν πέτραις</l>
+                <l n="908">αἰῶνα κωκύσουσιν ἠλοκισμένοι.</l>
+                <pb n="570" />
+                <l n="909">ἄλλην δʼ ἐπʼ ἄλλῃ κῆρα κινήσει θεός,</l>
+                <l n="910">λυγρὴν πρὸ νόστου συμφορὰν δωρούμενος. </l>
+                <l n="911">τὸν δʼ Αἰσάρου τε ῥεῖθρα καὶ βραχύπτολις</l>
+                <l n="912">Οἰνωτρίας γῆς κεγχρίνῃ βεβρωμένον</l>
+                <l n="913">Κρίμισα φιτροῦ δέξεται μιαιφόνον·</l>
+                <l n="914">αὐτὴ γὰρ ἄκραν ἄρδιν εὐθυνεῖ χεροῖν</l>
+                <l n="915">σάλπιγξ ἀποψάλλουσα Μαιώτην πλόκον· </l>
+                <l n="916">Δύρα παρʼ ὄχθαις ὅς ποτε φλέξας θρασὺν</l>
+                <l n="917">λέοντα ῥαιβῷ χεῖρας ὥπλισε Σεύθῃ</l>
+                <l n="918">δράκοντʼ ἀφύκτων γομφίων λυροκτύπῳ.</l>
+                <l n="919">Κρᾶθις δὲ τύμβους ὄψεται δεδουπότος,</l>
+                <l n="920">εὐρὰξ Ἀλαίου Παταρέως ἀνακτόρων, </l>
+                <l n="921">Ναύαιθος ἔνθα πρὸς κλύδωνʼ ἐρεύγεται.</l>
+                <l n="922">κτενοῦσι δʼ αὐτὸν Αὔσονες Πελλήνιοι</l>
+                <l n="923">βοηδρομοῦντα Λινδίων στρατηλάταις,</l>
+                <l n="924">οὓς τῆλε Θερμύδρου τε Καρπάθου τʼ ὀρῶν</l>
+                <l n="925">πλάνητας αἴθων Θρασκίας πέμψει κύων, </l>
+                <l n="926">ξένην ἐποικήσοντας ὀθνείαν χθόνα.</l>
+                <l n="927">ἐν δʼ αὖ Μακάλλοις σηκὸν ἔγχωροι μέγαν</l>
+                <l n="928">ὑπὲρ τάφων δείμαντες, αἰανῆ θεὸν</l>
+                <l n="929">λοιβαῖσι κυδανοῦσι καὶ θύσθλοις βοῶν.</l>
+                <pb n="572" />
+                <l n="930">ὁ δʼ ἱπποτέκτων Λαγαρίας ἐν ἀγκάλαις, </l>
+                <l n="931">ἔγχος πεφρικὼς καὶ φάλαγγα θουρίαν,</l>
+                <l n="932">πατρῷον ὅρκον ἐκτίνων ψευδώμοτον,</l>
+                <l n="933">ὃν ἀμφὶ μήλων τῶν δορικτήτων τάλας</l>
+                <l n="934">πύργων Κομαιθοῦς συμπεφυρμένων στρατῷ</l>
+                <l n="935">στεργοξυνεύνων οὕνεκεν νυμφευμάτων </l>
+                <l n="936">Ἀλοῖτιν ἔτλη τὴν Κυδωνίαν Θρασὼ</l>
+                <l n="937">ὁρκωμοτῆσαι τόν τε Κρηστώνης θεὸν</l>
+                <l n="938">Κανδάονʼ ἤ Μάμερτον ὁπλίτην λύκον,</l>
+                <l n="939">ὁ μητρὸς ἐντὸς δελφύος στυγνὴν μάχην</l>
+                <l n="940">στήσας ἀραγμοῖς πρὸς κασίγνητον χεροῖν, </l>
+                <l n="941">οὔπω τὸ Τιτοῦς λαμπρὸν αὐγάζων φάος</l>
+                <l n="942">οὐδʼ ἐκφυγὼν ὠδῖνας ἀλγεινὰς τόκων.</l>
+                <l n="943">τοιγὰρ πόποι φύξηλιν ἤνδρωσαν σπόρον,</l>
+                <l n="944">πύκτην μὲν ἐσθλόν, πτῶκα δʼ ἐν κλόνῳ δορός,</l>
+                <l n="945">καὶ πλεῖστα τέχναις ὠφελήσαντα στρατόν· </l>
+                <l n="946">ὃς ἀμφὶ Κῖριν καὶ Κυλιστάνου <note n="1" place="unspecified">*kulista/nou
+                    Scheer from E.M. 544. 30 *kulista/r(n)ou.</note>γάνος</l>
```

Comment:
should read

<note place="inline" xml:lang="eng"><foreign xml:lang="grc">Κυλιστάνουu</foreign> Scheer from E.M. 544. 30 <foreign xml:lang="grc">Κυλιστάρ(ν)ου</foreign>.</note>

Note beta code has not been converted in most notes in all of these texts.